integrations/slack/src/links.ts

```diff
@@ -43,7 +43,7 @@ export async function unfurlLink(event: LinkSharedSlackEvent, context: SlackRunt
     const installationApiClient = await api.createInstallationClient('slack', installation.id);
 
     // Resolve links to their content
-    const unfurls = {};
+    const unfurls: Record<string, any> = {};
```

Comment:
Maybe this would be better?
```suggestion
    const unfurls: Record<string, object> = {};
```